modules/legacy-geo/src/test/java/org/elasticsearch/legacygeo/GeoJsonShapeParserTests.java

```diff
@@ -344,8 +343,6 @@ public void testParsePolygon() throws IOException, ParseException {
         assertGeometryEquals(p, polygonGeoJson, false);
     }
 
-    @UpdateForV9(owner = UpdateForV9.Owner.SEARCH_ANALYTICS)
-    @AwaitsFix(bugUrl = "this test is using pre 8.0.0 index versions so needs to be removed or updated")
```

Comment:
This geo related tests and below are now functional again, and they should be kept in the codebase until v7 indices are supported.